lectures/cobweb.md

```diff
@@ -326,21 +326,21 @@ m = Market()
 plot45(m, 0, 9, 2, num_arrows=3)
 ```
 
-The plot shows the function $g$ defined in {eq}`def_g` and the $45$ degree line.
+The plot shows the function $g$ defined in {eq}`def_g` and the $45\degree$ line.
 
 Think of $ p_t $ as a value on the horizontal axis.
 
 Since $p_{t+1} = g(p_t)$, we use the graph of $g$ to see $p_{t+1}$ on the vertical axis.
 
 Clearly,
 
-- If $ g $ lies above the 45 degree line at $p_t$, then we have $ p_{t+1} > p_t $.
-- If $ g $ lies below the 45 degree line at $p_t$, then we have $ p_{t+1} < p_t $.
-- If $ g $ hits the 45 degree line at $p_t$, then we have $ p_{t+1} = p_t $, so $ p_t $ is a steady state.
+- If $ g $ lies above the 45-degree line at $p_t$, then we have $ p_{t+1} > p_t $.
+- If $ g $ lies below the 45-degree line at $p_t$, then we have $ p_{t+1} < p_t $.
+- If $ g $ hits the 45-degree line at $p_t$, then we have $ p_{t+1} = p_t $, so $ p_t $ is a {ref}` steady state <scalar-dynam:steady-state>`.
```

Comment:
Is this `steady state` link required here or is the `steady state` terminology clearly understood in this context?